### PR TITLE
traefik@1: disable

### PR DESCRIPTION
**Pull request by SMillerDev**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by SMillerDev** [comment]
Related to https://github.com/Homebrew/homebrew-core/pull/107165

File: Formula/traefik@1.rb [hunk 1]
```diff
@@ -19,6 +19,9 @@ class TraefikAT1 < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "69adbe00d348f14f732bbd9fe57d57e6f513f6f1c72f5ff2f4e09fae5b4ee2d9"
   end
 
+  # support ended 2021-12-31: https://doc.traefik.io/traefik/deprecation/releases/
+  deprecate! date: "2022-12-31", because: :unsupported
```
Comment (carlocab) [inline Formula/traefik@1.rb:23]:
```suggestion
  deprecate! date: "2022-08-03", because: :unsupported
```

Comment (carlocab) [reply Formula/traefik@1.rb:23]: Or:
```suggestion
  disable! date: "2022-12-31", because: :unsupported
```

Comment (SMillerDev) [reply Formula/traefik@1.rb:23]: Actually meant that to be
```suggestion
  disable! date: "2022-12-31", because: :unsupported
```

**Review by carlocab — APPROVED** [review]

**Pull request merged**